src/main/java/teammates/common/datatransfer/FeedbackSessionQuestionsBundle.java

```diff
@@ -105,7 +105,27 @@ public int compare(Map.Entry<String, String> o1, Map.Entry<String, String> o2) {
 
         return result;
     }
+    
+    /**
+     * Removes question from the bundle if the question has givers, recipients or responses that are anonymous to instructors.
```

Comment:
Actually is both. It's either the giver or recipient is anonymous or the response is hidden @damithc.  Will update the comments and method name?